src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__content {
+    margin-bottom: 16px;
+  }
+
+  &__image {
+    margin: 0 auto;
+    display: block;
+    max-height: 134px;
+    margin-bottom: 40px;
```

Comment:
Please don't forget add styles in correct order
https://codeguide.academy/html-css.html#css-order